modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -133,7 +133,7 @@ Prior to Redpanda version 24.2, this meant that some cores on a broker could ina
 
 Starting in v24.2, topic-aware intra-broker partition balancing allows for dynamically reassigning partitions within a broker.  Redpanda prioritizes an even distribution of a topic's partition replicas across all cores in a broker. If a broker's core count changes, when the broker starts back up, Redpanda can check partition assignments across the broker's cores and reassign partitions, so that a balanced assignment is maintained across all cores. Redpanda can also check partition assignments when partitions are added to or removed from a broker, and rebalance the remaining partitions between cores.
 
-To determine when to use intra-broker partition balancing, use the public metrics for CPU usage described in the xref:manage:monitoring.adoc#cpu-usage[Monitoring] guide, as well as the xref:reference:internal-metrics-reference.adoc#vectorized_reactor_utilization[`vectorized_reactor_utilization`] internal metric.
+To determine when to use intra-broker partition balancing, use the public metrics for CPU usage described in the xref:manage:monitoring.adoc#cpu-usage[Monitoring] guide, as well as the xref:reference:public-metrics-reference.adoc#redpanda_cpu_busy_seconds_total[`redpanda_cpu_busy_seconds_total`] public metric.
```

Comment:
So `redpanda_cpu_busy_seconds_total` is one of the "public metrics for CPU usage described in the Monitoring guide" as mentioned earlier in the sentence. Changing the name/link makes it redundant. @ztlpn could you confirm that we do want to keep the reference to `vectorized_reactor_utilization` here?

Reply:
Right, I guess we can just leave the link to the monitoring guide and remove the reference to the individual metric here.